packages/databricks-vscode/src/configuration/ConnectionManager.ts

```diff
@@ -175,16 +162,54 @@ export class ConnectionManager implements Disposable {
         return this._workspaceClient?.apiClient;
     }
 
+    async resolveAuth() {
+        const host = await this.configModel.get("host");
+        const target = this.configModel.target;
+        if (host === undefined || target === undefined) {
+            return;
+        }
+
+        // Try to load a profile user had previously selected for this target
+        const savedProfile = (
+            await this.configModel.overrideableConfigModel.load(["authProfile"])
+        )?.authProfile;
```

Comment:
Seems simpler to just load everything, since we are selecting authProfile from the object in any case.

Same for the authParams later. Or are there some hidden reason to do such partial loading? As I understand we are doing the full loading internally anyway

Reply:
Good catch. Left over from some previous commits.